packages/abi-coder/test/Interface.test.ts

```diff
@@ -53,7 +53,7 @@ const exhaustiveExamplesInterface = new Interface(exhaustiveExamplesAbi);
 /**
  * @group node
  */
-describe('Abi interface', () => {
+describe.skip('Abi interface', () => {
```

Comment:
Skipped as currently [blocked](https://github.com/FuelLabs/sway/issues/5860).

Reply:
Should we un-skip it after https://github.com/FuelLabs/sway/issues/5860#issuecomment-2072202247 and adjust the contract?

Reply:
Yep agreed, currently working through that at the moment 👍🏻

Reply:
I have it compiling and now fixing the interface tests.